pom.xml

```diff
@@ -5,8 +5,8 @@
     <modelVersion>4.0.0</modelVersion>
 
     <groupId>me.moomoo</groupId>
-    <artifactId>anarchyexploitfixes</artifactId>
-    <version>1.27.0</version>
```

Comment:
2.0.0

Reply:
yea this pr was made too early, the person was trying to create an issue, not a pr